Add sxhkd-mode-indentation-length var to set indentation length

jriboux opened the pull request:
Added `sxhkd-mode-indentation-length` var to set indentation length.

Renamed `sxhkd-mode-indentation-length` function to `sxhkd-mode-current-indentation-length`.

jriboux committed:
Add sxhkd-mode-indentation-length var to set indentation length

xFA25E commented:
Hi! Thanks for your work!

xFA25E commented on sxhkd-mode.el:
```diff
@@ -108,8 +111,8 @@ Can be t, nil or ask"
         (end-of-line)
-        (forward-char -1)
-        (if (looking-at-p (rx "\\")) 0 1)))))
+        (forward-char (* -1 sxhkd-mode-indentation-length))
```
Could you please change this line to `(- sxhkd-mode-indentation-length)`? Thanks!

jriboux replied:
Thanks for the review, it is fixed in last commit.

xFA25E commented on sxhkd-mode.el:
```diff
@@ -44,6 +44,9 @@ Can be t, nil or ask"
                 (const :tag "Ask" ask))
   :group 'sxhkd-mode)
 
+(defvar sxhkd-mode-indentation-length 2
+  "Indentation length")
```
This one should be defcustom:
```
(defcustom sxhkd-mode-indentation-length 2
  "Indentation length."
  :type 'integer
  :group 'sxhkd-mode)
```

jriboux replied:
Thanks for the review, it is fixed in last commit.

jriboux committed:
Code improvements

- Fixed sxhkd-mode-current-indentation-length declaration
- Code simplification
